README.md
# violet-lighting

changelog.txt
============ CHANGELOG =============

Version 5.2.4-1      2019-10-28
* Updated All in One SEO Pack to 3.2.10
* Updated MySQL to 8.0.18
* Updated PHP to 7.3.11
* Updated SQLite to 3.30.1.
* Updated WP Mail SMTP to 1.7.0

Version 5.2.4-0      2019-10-15
* Updated WordPress to 5.2.4

Version 5.2.3-1      2019-10-14
* Fix upload of big images
* Remove WordPress Multisite Domain Mapping plugin
* Updated All in One SEO Pack to 3.2.9
* Updated All-in-One WP Migration to 7.8
* Updated Amazon Polly to 3.1.5
* Updated Google Analytics plugin to 7.9.0
* Updated Jetpack to 7.8
* Updated lego to 3.1.0
* Updated OpenSSL to 1.1.1d
* Updated PHP to 7.3.10
* Updated phpMyAdmin to 4.9.1
* Updated Simple Tags to 2.5.5
* Updated SQLite to 3.30.0.
* Updated Varnish to 6.0.4
* Updated WP Mail SMTP to 1.6.2
* Updated Akismet Anti-Spam to 4.1.2

Version 5.2.3-0      2019-09-05
* Maintenance release
* Updated Apache to 2.4.41
* Updated lego to 3.0.2
* Updated OpenSSL to 1.1.1c
* Updated PHP to 7.3.9
* Updated WP-CLI to 2.3.0
* Updated WordPress to 5.2.3

Version 5.2.2-3      2019-07-25
* Updated lego to 2.7.1
* Updated MySQL to 8.0.17
* Updated Ghostscript to 9.05

Version 5.2.2-2      2019-07-17
* Maintenance release
* Updated PHP to 7.3.7
* Updated SQLite to 3.29.0.

Version 5.2.2-0      2019-06-19
* Updated WordPress to 5.2.2

Version 5.2.1-1      2019-06-19
* Fixed issue on Windows 10 build 1903
* Updated lego to 2.6.0
* Updated OpenSSL to 1.0.2s
* Updated PHP to 7.3.6
* Updated phpMyAdmin to 4.9.0.1

Version 5.2.1-0      2019-05-22
* Updated WordPress to 5.2.1

Version 5.2-1      2019-05-13
* Maintenance release
* Add Bitnami HTTPS Configuration Tool (bncert)

Version 5.2-0      2019-05-08
* Updated PHP to 7.3.5
* Updated WordPress to 5.2

Version 5.1.1-2      2019-04-30
* Maintenance release
* Updated lego to 2.5.0
* Updated MySQL to 8.0.16
* Updated PHP to 7.2.17
* Updated SQLite to 3.28.0
* Updated Varnish to 6.0.3
* Updated WP-CLI to 2.2.0

Version 5.1.1-1      2019-04-09
* Fix Apache Carpe Diem vulnerability CVE-2019-0211
* Updated Apache to 2.4.39
* Updated lego to 2.4.0

Version 5.1.1-0      2019-03-13
* Updated OpenSSL to 1.0.2r
* Updated PHP to 7.2.16
* Updated SQLite to 3.27.2
* Updated WordPress to 5.1.1

Version 5.1-0      2019-02-23
* Updated Apache to 2.4.38
* Updated lego to 2.2.0
* Updated MySQL to 8.0.15
* Updated PHP to 7.2.15
* Updated phpMyAdmin to 4.8.5
* Updated SQLite to 3.27.1
* Updated WordPress to 5.1

Version 5.0.3-2      2019-01-15
* Updated plugins and translations

Version 5.0.3-1      2019-01-11
* Maintenance release

Version 5.0.3-0      2019-01-10
* Updated SQLite to 3.26.0
* Updated WordPress to 5.0.3
* Updated WP-CLI to 2.1.0

Version 5.0.2-1      2018-12-21
* Updated WP-CLI to 2.0.1

Version 5.0.2-0      2018-12-20
* Updated WordPress to 5.0.2
* Updated WP-CLI to 2.1.0

Version 5.0.1-0      2018-12-13
* Updated WordPress to 5.0.1

Version 5.0-2      2018-12-12
* Fixed PHP apps installation with MySQL8
* Updated MySQL to 8.0.13
* Updated PHP to 7.2.13
* Updated phpMyAdmin to 4.8.4

Version 5.0-1      2018-12-07
* Updated PHP to 7.2.12

Version 5.0-0      2018-12-07
* Updated Apache to 2.4.37
* Updated ImageMagick to 6.9.8
* Updated lego to 1.2.1
* Updated MySQL to 5.7.24
* Updated OpenSSL to 1.0.2q
* Updated PHP to 7.1.24
* Updated phpMyAdmin to 4.8.3
* Updated SQLite to 3.18.0
* Updated Varnish to 4.1.0
* Updated WordPress to 5.0
* Updated WP-CLI to 2.0.1

Version 4.9.8-3      2018-11-29
* Reverted MySQL version to 5.7.23 to avoid antivirus reports with 5.7.24 on Windows
* Updated lego to 1.2.1
* Updated OpenSSL to 1.0.2q
* Updated PHP to 7.1.24

Version 4.9.8-2      2018-10-29
* Fixed SpectreRSB Vulnerability (CVE-2018-15572)
* Updated Apache to 2.4.37
* Updated lego to 1.1.0
* Updated MySQL to 5.7.24
* Updated PHP to 7.1.23

Version 4.9.8-1      2018-09-17
* Added Lego tool for configuring Let's encrypt with Apache
* Updated OpenSSL to 1.0.2p
* Updated phpMyAdmin to 4.8.3
* Updated WP-CLI to 2.0.1
* Updated lego to 1.0.1

Version 4.9.8-0      2018-08-03
* Updated Apache to 2.4.34
* Updated MySQL to 5.7.23
* Updated PHP to 7.0.31
* Updated WordPress to 4.9.8

Version 4.9.7-0      2018-07-05
* Updated PageSpeed to 1.13.35.2
* Updated phpMyAdmin to 4.8.2
* Updated WordPress to 4.9.7

Version 4.9.6-0      2018-05-18
* Updated WordPress to 4.9.6

Version 4.9.5-2      2018-05-08
* Updated MySQL to 5.7.22
* Updated PHP to 7.0.30
* Updated SQLite to 3.18.0

Version 4.9.5-1      2018-04-28
* Updated WordPress configuration file to avoid FTP credentials issues
* Updated PHP to 7.0.29
* Updated phpMyAdmin to 4.8.0.1
* Updated WordPress to 4.9.5
* Updated WP-CLI to 1.5.1

Version 4.9.5-0      2018-04-04
* Updated Apache to 2.4.33
* Updated OpenSSL to 1.0.2o
* Updated WordPress to 4.9.5

Version 4.9.4-6      2018-03-13
* Updated DHCP package in Virtual Machines and cloud images: CVE-2018-5732
* Updated PHP to 7.0.28

Version 4.9.4-5      2018-03-06
* Updated ImageMagick to 6.9.8
* Updated phpMyAdmin to 4.7.9

Version 4.9.4-4      2018-02-16
* Updated metadata of the Amazon Polly plugin

Version 4.9.4-3      2018-02-15
* Updated plugins and translations
* Included Amazon Polly plugin in the AWS images

Version 4.9.4-2      2018-02-13
* Updated plugins and translations

Version 4.9.4-1      2018-02-09
* Generate the SSL certificates at installation time
(boot time in the cloud images and virtual machines)

Version 4.9.4-0      2018-02-07
* Updated WordPress to 4.9.4
* Updated WP-CLI to 1.5.0

Version 4.9.2-1      2018-01-26
* Updated MySQL to 5.7.21
* Updated PHP to 7.0.27
* Updated WP-CLI to 1.4.1

Version 4.9.2-0      2018-01-17
* Updated WordPress to 4.9.2

Version 4.9.1-2      2018-01-12
* Improved OS X VM images

Version 4.9.1-1      2018-01-11
* Updated OpenSSL to 1.0.2n
* Updated PHP to 7.0.26
* Updated phpMyAdmin to 4.7.7

Version 4.9.1-0      2017-11-30
* Updated WordPress to 4.9.1

Version 4.9-1      2017-11-29
* Added support for OS X VM

Version 4.9-0      2017-11-16

apps/wordpress/htdocs/wp-content/themes/neve/header.php
<?php
/**
 * The template for displaying the header
 *
 * Displays all of the head element and everything up until the page header div.
 *
 * @package Neve
 * @since   1.0.0
 */

$header_classes = apply_filters( 'nv_header_classes', 'header' );
?><!DOCTYPE html>
<html <?php language_attributes(); ?>>

<head>
	<meta charset="<?php bloginfo( 'charset' ); ?>">
	<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
	<link rel="profile" href="http://gmpg.org/xfn/11">
	<?php if ( is_singular() && pings_open( get_queried_object() ) ) : ?>
		<link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>">
	<?php endif; ?>
	<?php wp_head(); ?>
</head>

<body <?php body_class(); ?> <?php echo wp_kses( apply_filters( 'neve_body_data_attrs', '' ), array( '[class]' => true ) ); ?>>
<?php wp_body_open(); ?>
<div class="wrapper">
	<header class="<?php echo esc_attr( $header_classes ); ?>" role="banner">
	<h1>Test</h1>
		<a class="neve-skip-link show-on-focus" href="#content" tabindex="0">
			<?php echo __( 'Skip to content', 'neve' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
		</a>
		<?php
		neve_before_header_trigger();
		if ( apply_filters( 'neve_filter_toggle_content_parts', true, 'header' ) === true ) {
			do_action( 'neve_do_header' );
		}
		neve_after_header_trigger();
		?>
	</header>
	<?php do_action( 'neve_before_primary' ); ?>

	<main id="content" class="neve-main" role="main">

<?php
do_action( 'neve_after_primary_start' );



apps/phpmyadmin/htdocs/tmp/twig/65/650af88591c31d1a214e989419f5395e29402c995f5f1872c9e08dea065b633f.php
<?php

use Twig\Environment;
use Twig\Error\LoaderError;
use Twig\Error\RuntimeError;
use Twig\Markup;
use Twig\Sandbox\SecurityError;
use Twig\Sandbox\SecurityNotAllowedTagError;
use Twig\Sandbox\SecurityNotAllowedFilterError;
use Twig\Sandbox\SecurityNotAllowedFunctionError;
use Twig\Source;
use Twig\Template;

/* login/footer.twig */
class __TwigTemplate_3e093f70a61099c36de9ffbd4ff5b0ae0ee8ceba69f3f9670b7623b70664d844 extends \Twig\Template
{
    public function __construct(Environment $env)
    {
        parent::__construct($env);

        $this->parent = false;

        $this->blocks = [
        ];
    }

    protected function doDisplay(array $context, array $blocks = [])
    {
        // line 1
        echo "</div>
";
    }

    public function getTemplateName()
    {
        return "login/footer.twig";
    }

    public function getDebugInfo()
    {
        return array (  30 => 1,);
    }

    /** @deprecated since 1.27 (to be removed in 2.0). Use getSourceContext() instead */
    public function getSource()
    {
        @trigger_error('The '.__METHOD__.' method is deprecated since version 1.27 and will be removed in 2.0. Use getSourceContext() instead.', E_USER_DEPRECATED);

        return $this->getSourceContext()->getCode();
    }

    public function getSourceContext()
    {
        return new Source("", "login/footer.twig", "C:\\Bitnami\\wordpress-5.2.4-1\\apps\\phpmyadmin\\htdocs\\templates\\login\\footer.twig");
    }
}


apps/phpmyadmin/htdocs/tmp/twig/7e/7ef44d0dfdd0494e594a9b4a9b0cc341ec795aad23fdab013353fa36bc3a411f.php
<?php

use Twig\Environment;
use Twig\Error\LoaderError;
use Twig\Error\RuntimeError;
use Twig\Markup;
use Twig\Sandbox\SecurityError;
use Twig\Sandbox\SecurityNotAllowedTagError;
use Twig\Sandbox\SecurityNotAllowedFilterError;
use Twig\Sandbox\SecurityNotAllowedFunctionError;
use Twig\Source;
use Twig\Template;

/* login/header.twig */
class __TwigTemplate_b73de67cfc3c3b7aee03380b8c2fd0a8c22c78411ee50528258d2126036cfc72 extends \Twig\Template
{
    public function __construct(Environment $env)
    {
        parent::__construct($env);

        $this->parent = false;

        $this->blocks = [
        ];
    }

    protected function doDisplay(array $context, array $blocks = [])
    {
        // line 1
        echo "<div class=\"container\">
<a href=\"";
        // line 2
        echo twig_escape_filter($this->env, PhpMyAdmin\Core::linkURL("https://www.phpmyadmin.net/"), "html", null, true);
        echo "\" target=\"_blank\" rel=\"noopener noreferrer\" class=\"logo\">
<img src=\"";
        // line 3
        echo twig_escape_filter($this->env, $this->getAttribute(($context["theme"] ?? null), "getImgPath", [0 => "logo_right.png", 1 => "pma_logo.png"], "method"), "html", null, true);
        echo "\" id=\"imLogo\" name=\"imLogo\" alt=\"phpMyAdmin\" border=\"0\" />
</a>
<h1>";
        // line 5
        echo sprintf(_gettext("Welcome to %s"), "<bdo dir=\"ltr\" lang=\"en\">phpMyAdmin</bdo>");
        echo "</h1>

<noscript>
";
        // line 8
        echo call_user_func_array($this->env->getFunction('Message_error')->getCallable(), [_gettext("Javascript must be enabled past this point!")]);
        echo "
</noscript>

<div class=\"hide\" id=\"js-https-mismatch\">
";
        // line 12
        echo call_user_func_array($this->env->getFunction('Message_error')->getCallable(), [_gettext("There is mismatch between HTTPS indicated on the server and client. This can lead to non working phpMyAdmin or a security risk. Please fix your server configuration to indicate HTTPS properly.")]);
        echo "
</div>
";
    }

    public function getTemplateName()
    {
        return "login/header.twig";
    }

    public function isTraitable()
    {
        return false;
    }

    public function getDebugInfo()
    {
        return array (  55 => 12,  48 => 8,  42 => 5,  37 => 3,  33 => 2,  30 => 1,);
    }

    /** @deprecated since 1.27 (to be removed in 2.0). Use getSourceContext() instead */
    public function getSource()
    {
        @trigger_error('The '.__METHOD__.' method is deprecated since version 1.27 and will be removed in 2.0. Use getSourceContext() instead.', E_USER_DEPRECATED);

        return $this->getSourceContext()->getCode();
    }

    public function getSourceContext()
    {
        return new Source("", "login/header.twig", "C:\\Bitnami\\wordpress-5.2.4-1\\apps\\phpmyadmin\\htdocs\\templates\\login\\header.twig");
    }
}


apps/phpmyadmin/htdocs/tmp/twig/d6/d6f26bd99abdf5fb673f8d2f84e6387554e6d53a3f871aaaeea2b1caad8cfd57.php
<?php

use Twig\Environment;
use Twig\Error\LoaderError;
use Twig\Error\RuntimeError;
use Twig\Markup;
use Twig\Sandbox\SecurityError;
use Twig\Sandbox\SecurityNotAllowedTagError;
use Twig\Sandbox\SecurityNotAllowedFilterError;
use Twig\Sandbox\SecurityNotAllowedFunctionError;
use Twig\Source;
use Twig\Template;

/* select_lang.twig */
class __TwigTemplate_f547b0b5492ee4be86f483d30b6e9986183de305ceac4ef1db0e94a2ae00f745 extends \Twig\Template
{
    public function __construct(Environment $env)
    {
        parent::__construct($env);

        $this->parent = false;

        $this->blocks = [
        ];
    }

    protected function doDisplay(array $context, array $blocks = [])
    {
        // line 1
        echo "    <form method=\"get\" action=\"index.php\" class=\"disableAjax\">
    ";
        // line 2
        echo PhpMyAdmin\Url::getHiddenInputs(($context["_form_params"] ?? null));
        echo "

    ";
        // line 4
        if (($context["use_fieldset"] ?? null)) {
            // line 5
            echo "        <fieldset>
            <legend lang=\"en\" dir=\"ltr\">";
            // line 6
            echo ($context["language_title"] ?? null);
            echo "</legend>
    ";
        } else {
            // line 8
            echo "        <bdo lang=\"en\" dir=\"ltr\">
            <label for=\"sel-lang\">";
            // line 9
            echo ($context["language_title"] ?? null);
            echo "</label>
        </bdo>
    ";
        }
        // line 12
        echo "
    <select name=\"lang\" class=\"autosubmit\" lang=\"en\" dir=\"ltr\" id=\"sel-lang\">

    ";
        // line 15
        $context['_parent'] = $context;
        $context['_seq'] = twig_ensure_traversable(($context["available_languages"] ?? null));
        foreach ($context['_seq'] as $context["_key"] => $context["language"]) {
            // line 16
            echo "        ";
            // line 17
            echo "        <option value=\"";
            echo twig_escape_filter($this->env, twig_lower_filter($this->env, $this->getAttribute($context["language"], "getCode", [], "method")), "html", null, true);
            echo "\"";
            // line 18
            if ($this->getAttribute($context["language"], "isActive", [], "method")) {
                // line 19
                echo "                selected=\"selected\"";
            }
            // line 21
            echo ">
        ";
            // line 22
            echo $this->getAttribute($context["language"], "getName", [], "method");
            echo "
        </option>
    ";
        }
        $_parent = $context['_parent'];
        unset($context['_seq'], $context['_iterated'], $context['_key'], $context['language'], $context['_parent'], $context['loop']);
        $context = array_intersect_key($context, $_parent) + $_parent;
        // line 25
        echo "
    </select>

    ";
        // line 28
        if (($context["use_fieldset"] ?? null)) {
            // line 29
            echo "        </fieldset>
    ";
        }
        // line 31
        echo "
    </form>
";
    }

    public function getTemplateName()
    {
        return "select_lang.twig";
    }

    public function isTraitable()
    {
        return false;
    }

    public function getDebugInfo()
    {
        return array (  100 => 31,  96 => 29,  94 => 28,  89 => 25,  80 => 22,  77 => 21,  74 => 19,  72 => 18,  68 => 17,  66 => 16,  62 => 15,  57 => 12,  51 => 9,  48 => 8,  43 => 6,  40 => 5,  38 => 4,  33 => 2,  30 => 1,);
    }

    /** @deprecated since 1.27 (to be removed in 2.0). Use getSourceContext() instead */
    public function getSource()
    {
        @trigger_error('The '.__METHOD__.' method is deprecated since version 1.27 and will be removed in 2.0. Use getSourceContext() instead.', E_USER_DEPRECATED);

        return $this->getSourceContext()->getCode();
    }

    public function getSourceContext()
    {
        return new Source("", "select_lang.twig", "C:\\Bitnami\\wordpress-5.2.4-1\\apps\\phpmyadmin\\htdocs\\templates\\select_lang.twig");
    }
}
